types/cli-progress/index.d.ts

```diff
@@ -82,6 +83,9 @@ export interface Options {
      */
     hideCursor?: boolean | null | undefined;
 
+    /** glue sequence (control chars) between bar elements (default: '') */
+    barGlue?: string;
```

Comment:
```suggestion
    barGlue?: string | undefined;
```
this module supports explicit undefined as values, so we advice to add exact optional property support here:
https://github.com/npkgz/cli-progress/blob/master/lib/options.js#L2
https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#exact-optional-property-types